app/models/question.rb

```diff
@@ -20,6 +34,26 @@ class Question < ActiveRecord::Base
 
   validates :text, presence: true
   validates :number, presence: true, numericality: { only_integer: true }
-
   validates :question_type, presence: true, inclusion: {in: %w[short_answer multiple_choice true_false]}
+  validates_with QuestionOrdersValidator, on: :next_question
+
+  def formatted_question_and_responses
+    # TODO (Chris) will implement later since it's pretty complex
+    # involves getting the responses too
+    text
+  end
+
+  # Assumes the response is valid
+  # Assumes the question orders is valid for this question valid?(:next_question) has been checked
+  def next_question(response)
+    if question_type == 'short_answer'
```

Comment:
Going to check if the question has response choices to determine the type of question, because it seems cleaner than `question_orders.first.is_a DefaultQuestionOrder` which was the other way I was thinking of doing this.